app/components/error_flash_component.rb

```diff
@@ -0,0 +1,11 @@
+class ErrorFlashComponent < ActionView::Component::Base
+  validates :message, presence: true
+
+  def initialize(message:)
+    @message = message
+  end
+
+private
+
+  attr_reader :message
+end
```

Comment:
Could the flash component code be moved to a separate PR?

Feels like this could be a separate card and would be nicer in it's own PR to get Design feedback.

Reply:
I've already asked for design feedback on this specific story/PR. If they say it's ok I'll keep the SuccessFlashComponent because the story requires this feedback at the last stage. I'll prepare a separate PR anyway implementing different severity levels, anyway.